packages/react-core/src/components/Dropdown/DropdownWithContext.tsx

```diff
@@ -82,7 +100,35 @@ export class DropdownWithContext extends React.Component<DropdownProps & OUIAPro
       <DropdownContext.Consumer>
         {({ baseClass, baseComponent, id: contextId }) => {
           const BaseComponent = baseComponent as any;
-          return (
+          const menuComponent = (
+            <DropdownMenu
+              setMenuComponentRef={this.setMenuComponentRef}
+              component={component}
+              isOpen={isOpen}
+              position={position}
+              aria-labelledby={contextId ? `${contextId}-toggle` : id}
+              openedOnEnter={openedOnEnter}
+              isGrouped={isGrouped}
+              autoFocus={openedOnEnter && autoFocus}
+            >
+              {renderedContent}
+            </DropdownMenu>
+          );
+          const popoverContent = (
+            <div
+              className={css(
+                baseClass,
+                direction === DropdownDirection.up && styles.modifiers.top,
+                position === DropdownPosition.right && styles.modifiers.alignRight,
+                isOpen && styles.modifiers.expanded,
+                className
+              )}
+              style={{ position: 'absolute' }}
```

Comment:
is the plan to add css in core for this?  Or is this a temporary solution?

Reply:
Tippy has its own set of css and overrides on that css in react-styles which is outside the realm of PF core. I could add a class there that targets the dropdown element, but might be more trouble than it's worth

Reply:
What if we added a modifier to the `.pf-c-dropdown` container that allows us to target specific styles when it's used in this context (only the `.pf-c-dropdown__menu` item is inside of it), including adding this style and anything else that might come up.

FWIW, `position: absolute` fixes an alignment issue due to `.pf-c-dropdown` being `inline-block`, and then the `.pf-c-dropdown__menu` element inside of it is absolutely positioned relative to `.pf-c-dropdown`. The default alignment for an inline block element is `vertical-align: bottom;`, so it's positioning it in the bottom of the containing block. You could also fix it by adding `vertical-align: top;` to `.pf-c-dropdown`, but if the dropdown isn't being displayed inline with content, using `display: block` is probably the simplest fix here. I made a quick demo that shows the problem.  https://codepen.io/mcoker/pen/ZEQbyyY

Reply:
Hey @mcoker , thanks for taking a look! I was curious why the menu was offset unless i added absolute positioning to it. Seems like there are a variety of ways to solve this. Is it fine to keep this fix (position: absolute) temporarily until we have a core modifier class?

Reply:
using `display: block` as the temporary solution until https://github.com/patternfly/patternfly/issues/3156 can be brought into react